### PR TITLE
add bouncing ball to the project

### PR DESCRIPTION
**Pull request by shrlaw**
Please add my animation.

### CODE REVIEW

File: Art/shrlaw-bouncing/includes.js
Comment (LaurelineP) [inline Art/shrlaw-bouncing/includes.js:1]: ## No JS allowed 
I noticed you've dubplicated the animation-nation includes.js - there is no need 
you can remove this file in your folder

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha again @shrlaw 
Thanks for your new contribution,
Please address the given feedback so we could proceed in merging your PR :)

**Comment by shrlaw** [comment]
I made the requested change.

File: Art/shrlaw-bouncing/includes.js
Comment (shrlaw) [reply Art/shrlaw-bouncing/includes.js:1]: I removed includes.js.

**Pull request merged**